crates/sui-sdk/tests/tests.rs

```diff
@@ -32,3 +44,65 @@ fn keystore_display_test() -> Result<(), anyhow::Error> {
     assert!(!keystore.to_string().contains("keys:"));
     Ok(())
 }
+
+#[tokio::test]
+async fn test_verify_signature() {
+    let (address, sec1): (_, Ed25519KeyPair) = get_key_pair();
```

Comment:
please test failures as well here and below

Reply:
the more granular test cases are already included as part of the underlying authenticator test. included a simple negative here.